config/core/config/kustomization.yaml

```
@@ -0,0 +1,2 @@
+resources:
+  - ingress-domain.yaml
```

Also need to rename the target file.

```suggestion
  - settings.yaml
```